### PR TITLE
[homework #6] - Js basics 1 by Federico Benitez

### PR DESCRIPTION
**Pull request by Federico-Benitez**
Exercises for js basics 1 without exercise n°16

### CODE REVIEW

File: homeworks/federico-benitez/06-js-basics-1/index.js [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+
+  const middle = s.length / 2;
+  return `${s[middle - 1]}${s[middle]}`;
+}
+
+/**
+ * Exercise 7
+ * http://www.codewars.com/kata/partition-on
+ */
+function partitionOn(pred, items) {
```
Comment (VitalikMelnyk) [inline homeworks/federico-benitez/06-js-basics-1/index.js:156]: Not passed all tests. Please verify
Screenshot 2022-08-17 at 19 17 01

File: homeworks/federico-benitez/06-js-basics-1/index.js [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+ * TODO: resolve
+ */
+
+/* ----- 404 not found ---- */
+
+/**
+ * Exercise 9 - Find the odd int
+ * https://www.codewars.com/kata/find-the-odd-int/
+ */
+function findOdd(A) {
```
Comment (VitalikMelnyk) [inline homeworks/federico-benitez/06-js-basics-1/index.js:182]: Not passed all tests. Please, verify.
Screenshot 2022-08-17 at 19 18 27

File: homeworks/federico-benitez/06-js-basics-1/index.js [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+
+  return fib[n - 1];
+}
+
+/**
+ * Exercise 14 - Cat and Mouse - 2D Version
+ * https://www.codewars.com/kata/cat-and-mouse-2d-version/
+ */
+
+function catMouse(map, moves) {
```
Comment (VitalikMelnyk) [inline homeworks/federico-benitez/06-js-basics-1/index.js:269]: Not passed all tests. Please, verify.
Screenshot 2022-08-17 at 19 21 08

File: homeworks/federico-benitez/06-js-basics-1/index.js [hunk 4]
```diff
@@ -0,0 +1,390 @@
...
+      };
+    }
+  });
+  return cords;
+}
+
+/**
+ * Exercise 15 - Duplicate Encoder
+ * https://www.codewars.com/kata/duplicate-encoder
+ */
```
Comment (VitalikMelnyk) [inline homeworks/federico-benitez/06-js-basics-1/index.js:304]: Not passed all tests. Please, verify.
Screenshot 2022-08-17 at 19 24 38

**Comment by VitalikMelnyk** [comment]
Hello @Federico-Benitez.
I have checked your homework. Please, check my notes.
Thanks!

**Review by solvd-mmelnyk — APPROVED** [review]

**Pull request merged**